src/compressor.rs

```diff
@@ -1,22 +1,46 @@
+/// Custom compression method
+///
+/// ```
+/// # use rusty_leveldb::{Compressor, CompressorId};
+/// 
+/// #[derive(Debug, Clone, Copy, Default)]
+/// pub struct CustomCompressor;
+///
+/// impl CompressorId for CustomCompressor {
+///     const ID: u8 = 0;
```

Comment:
a small detail: given that `0` is for the "none" compressor, maybe use an obviously bogus number like "42"

Reply:
Oh I forget to change that. I ctrl-c ctrl-v that piece of code from NoneCompressor. 😅